glean-core/rlb/src/dispatcher/global.rs

```diff
@@ -129,4 +129,42 @@ mod test {
         expected.push(200);
         assert_eq!(&*result.lock().unwrap(), &expected);
     }
+
+    #[test]
+    #[ignore] // We can't reset the queue at the moment, so flushing it breaks other tests.
```

Comment:
Is this still true? We do have `reset_dispatcher()` now!

Reply:
I guess it's not the way it is written, but because of multi-threaded tests this then needs to hold the global lock.
IMO we can do that in a followup (e.g. https://bugzilla.mozilla.org/show_bug.cgi?id=1675476)